lib/livebook_app.ex

```diff
@@ -8,8 +8,9 @@ if Mix.target() == :app do
 
     @impl true
     def init(_) do
-      {:ok, _} = ElixirKit.start()
-      {:ok, nil}
+      {:ok, pid} = ElixirKit.start()
+      ref = Process.monitor(pid)
```

Comment:
this Process.monitor could happen inside ElixirKit.start() though so people always handle it, if you prefer.

Reply:
I would make it explicit like this. Ship it!